param/parameterized.py

```diff
@@ -494,7 +494,7 @@ def _params_depended_on(minfo):
 
 
 def _m_caller(self, n):
-    def caller(event):
+    def caller(*events):
```

Comment:
Did this need to change? `event` isn't even used so why does it matter if it is `event` or `*events`? (other than the semantically better pluralization)

Can anyone supply a docstring for this function?

Reply:
> Did this need to change? 

Because now you can have more than one event? (untested, as was comical code you spotted earlier)

Reply:
That is what we suspected... :-)